numpy/core/tests/test_multiarray.py

```diff
@@ -1860,16 +1842,19 @@ def test_sort(self):
         msg = 'test empty array sort with axis=None'
         assert_equal(np.sort(a, axis=None), a.ravel(), msg)
 
+    def test_sort_bad_ordering(self):
         # test generic class with bogus ordering,
         # should not segfault.
         class Boom:
             def __lt__(self, other):
                 return True
 
-        a = np.array([Boom()]*100, dtype=object)
+        a = np.array([Boom()] * 100, dtype=object)
         for kind in self.sort_kinds:
             msg = "bogus comparison object sort, kind=%s" % kind
```

Comment:
Could eliminate most of this message, again the test name tells the story
```suggestion
            msg = "kind=%s" % kind
```